### PR TITLE
Set runtime to Java 8

### PR DESCRIPTION
**Pull request by alexey-rise**
@alex-deaconu please review.

### CODE REVIEW

File: war/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-	<application>rvauser2</application>
-	<version>1-11-004</version>
+	<runtime>java8</runtime>
+	<service>default</service>
```
Comment (alex-deaconu) [inline war/WEB-INF/appengine-web.xml]: @alexey-rise as the RVA is deployed manually (from Eclipse) aren't the `application` and `version` fields still needed? Or are they deprecated for Java 8? Thanks.

**Comment by alex-deaconu** [comment]
@alexey-rise the RVA uses the org.restlet package to contact the Core. Do you think that requires the <url-stream-handler>? Thanks.

File: war/WEB-INF/appengine-web.xml (hunk 1 above)
Comment (alexey-rise) [reply war/WEB-INF/appengine-web.xml]: @alex-deaconu they are deprecated in the latest app engine SDK. 
Values are still needed, but you pass them as parameters when you run `gcloud` utility (or maven) to deploy.

**Comment by alexey-rise** [comment]
@alex-deaconu I am not sure I follow the last question. Can you elaborate?

**Comment by alex-deaconu** [comment]
@alexey-rise oops, the < .. > tag didn't show up properly. I was just wondering if org.restlet uses `HttpURLConnection` in the background, and if we need to fix that in the xml file. Thanks.

**Comment by alex-deaconu** [comment]
@alexey-rise I've updated the PR to re-include the Application/Version tags in appengine-web.xml. Without them, the Eclipse Plugin does not deploy correctly. 

Staged this PR at http://1-12-000.rvauser2.appspot.com/. I don't see any issues, @Rise-Vision/apps wondering if you could double check as well.

Thanks!

**Comment by fjvallarino** [comment]
@alex-deaconu looks good!

**Pull request merged**